practice-two/src/constants/messages.js

```diff
@@ -0,0 +1,18 @@
+const ERROR_MESSAGE = {
```

Comment:
should group by messages
examples:
MESSAGES = {
ERRORS: {},
SUCCESS: {}
};

Reply:
I changed as you commented
![image](https://github.com/datdragon777/React-Training/assets/74806826/1736d07e-7e81-4672-8090-d4216e603882)